config/redirects

```diff
@@ -1,43 +1,17 @@
 define: base https://docs.mongodb.com/bi-connector
-define: versions v1.1 v2.0 v2.1 v2.2 v2.3 v2.4 v2.5 v2.6 v2.7 v2.8 v2.9 v2.10 v2.11 v2.12 v2.13 v2.14 master
-symlink: v2.14 -> master
-symlink: upcoming -> master
-symlink: current -> master
+define: versions v2.10 v2.11 v2.12 v2.13 v2.14 master
 
-(v1.1-*]: bi-connector/${version}/reference/mongobiuser -> ${base}/${version}/components
-(v1.1-*]: bi-connector/${version}/reference/mongobischema -> ${base}/${version}/components
-[*-v1.1]: bi-connector/${version}/reference/mongosqld -> ${base}/${version}/components
-raw: bi-connector/ -> ${base}/current
-[*-v1.1]: bi-connector/${version}/tutorial/connecting -> ${base}/${version}/
-raw: bi-connector/schema-configuration -> ${base}/master/schema-configuration
-raw: bi-connector/installation -> ${base}/master/installation
-raw: bi-connector/components -> ${base}/master/components
-raw: bi-connector/components/mongobiuser -> ${base}/master/components
-raw: bi-connector/components/mongodrdl -> ${base}/master/components/mongodrdl
-raw: bi-connector/components/mongobischema -> ${base}/master/components
```

Comment:
[question]

Previously we had these two redirects:

1. `(v1.1-*]: bi-connector/${version}/reference/mongobischema -> ${base}/${version}/components`
2. `raw: bi-connector/components/mongobischema -> ${base}/master/components`

Now, we have:

` raw: bi-connector/components/mongobischema -> ${base}/master/components/`

Is that correct? Or should we retain two `mongobischema` redirects?